LoRaEngine/example.env

```diff
@@ -24,6 +24,8 @@ EDGEHUB_ROUTE='FROM /* INTO $upstream'
 PKT_FWD_VERSION=1.0.0
 #SPI DEV version information. Only needed for x86 gateways. Possible values are 1 or 2.
 PKT_FWD_SPI_DEV=2
+#set a custom speed for your SPI 
```

Comment:
I would add for clarity:
#set custom speed for your SPI or, if not needed, comment or remove PKT_FWD_SPI_DEV.